src/assets.js

```diff
@@ -55,7 +55,7 @@
     var name = url.replace('.' + getExtension(url), '');
 
     // remove slash if there is no folder in the path
-    return (name.indexOf('/') == 0 && name.lastIndexOf('/') == 0 ? name.substr(1) : name);
+    return (name.indexOf('/') === 0 && name.lastIndexOf('/') === 0 ? name.substr(1) : name);
```

Comment:
`indexOf` can only return a number so the triple equals isn't needed (to save bytes).